sw/device/silicon_creator/lib/shutdown_unittest.cc

```diff
@@ -491,12 +503,10 @@ TEST_F(ShutdownTest, InitializeRma) {
 TEST_F(ShutdownTest, RedactPolicyManufacturing) {
   // Devices in manufacturing or RMA states should not redact errors regardless
   // of the redaction level set by OTP.
-  constexpr auto kManufacturingStates = std::array<lifecycle_state_t, 10>{
-      kLcStateRaw,           kLcStateTestUnlocked0,
-      kLcStateTestUnlocked1, kLcStateTestUnlocked2,
-      kLcStateTestUnlocked3, kLcStateTestUnlocked4,
-      kLcStateTestUnlocked5, kLcStateTestUnlocked6,
-      kLcStateTestUnlocked7, kLcStateRma};
+  constexpr auto kManufacturingStates = std::array<lifecycle_state_t, 9>{
```

Comment:
nit: Should we use the type instead of `auto` for the sake of consistency?

Reply:
done